client/extensions/wp-job-manager/components/settings/job-submission/index.jsx

```diff
@@ -1,5 +1,224 @@
+								name="canEdit"
+								text="Allow editing of pending listings" />
+							<FormSettingExplanation isIndented>
+								{ translate( 'Users can continue to edit pending listings until they are approved by an admin.' ) }
+							</FormSettingExplanation>
+						</FormFieldset>
+					</Card>
+				</FormSection>
+			</form>
+
+			<form>
+				<FormSection name="duration">
+					<SectionHeader label={ translate( 'Listing Duration' ) }>
+						<FormButton compact
+							disabled={ isDisabled } />
+					</SectionHeader>
+					<Card>
+						<FormFieldset>
+							{ translate(
+								'Display listings for {{days /}} day',
+								'Display listings for {{days /}} days',
+								{
+									count: submissionDuration,
+									components: {
+										days:
+											<ReduxFormTextInput
+												disabled={ isDisabled }
+												min="0"
+												name="submissionDuration"
+												step="1"
+												type="number" />
+									}
+								}
+							) }
+							<FormSettingExplanation>
+								{ translate( 'Listings will display for the set number of days, then expire. ' +
+									'Leave this field blank if you don\'t want listings to have an expiration date.' ) }
+							</FormSettingExplanation>
+						</FormFieldset>
+					</Card>
+				</FormSection>
+			</form>
+
+			<form>
+				<FormSection name="method">
+					<SectionHeader label={ translate( 'Application Method' ) }>
+						<FormButton compact
+							disabled={ isDisabled } />
+					</SectionHeader>
+					<Card>
+						<FormFieldset>
+							<FormSettingExplanation>
+								{ translate( 'Choose the ways applicants can apply to job listings. Website URLs ' +
+									'will display as "Apply" links in job listings, and email addresses will display as-is.' ) }
+							</FormSettingExplanation>
+							<FormLabel>
+								<ReduxFormRadio
+									disabled={ isDisabled }
+									name="applicationMethod"
+									value="" />
+								<span>
+									{ translate( 'Email address or website URL' ) }
+								</span>
+							</FormLabel>
+
+							<FormLabel>
+								<ReduxFormRadio
+									disabled={ isDisabled }
+									name="applicationMethod"
+									value="email" />
+								<span>
+									{ translate( 'Email addresses only' ) }
+								</span>
+							</FormLabel>
+
+							<FormLabel>
+								<ReduxFormRadio
+									disabled={ isDisabled }
+									name="applicationMethod"
+									value="url" />
+								<span>
+									{ translate( 'Website URLs only' ) }
+								</span>
+							</FormLabel>
+						</FormFieldset>
+					</Card>
+				</FormSection>
+			</form>
+		</div>
+	);
+};
+
+JobSubmission.propTypes = {
+	isDisabled: PropTypes.bool,
+	translate: PropTypes.func,
 };
 
-export default JobSubmission;
+const connectComponent = connect(
+	( state ) => {
+		const selector = formValueSelector( 'submission', () => state.extensions.wpJobManager.form );
+
+		return {
+			submissionDuration: selector( state, 'duration.submissionDuration' ),
```

Comment:
It's because we're using a non-default location for storing the form state. From the [docs](http://redux-form.com/6.0.0-alpha.13/docs/api/FormValueSelector.md/):

> If you are using the `getFormState()` config parameter to keep the `redux-form` reducer
> at a place in your Redux store other than `state.form`, you must also provide the same
> function here to get the slice of Redux store where the `redux-form` reducer is mounted.
> Defaults to `state => state.form`.